src/Libraries/Nop.Services/ExportImport/Help/LocalizedPropertyByName.cs

```diff
@@ -0,0 +1,245 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc.Rendering;
+
+namespace Nop.Services.ExportImport.Help
+{
+    /// <summary>
+    /// A helper class to access the property by name
+    /// </summary>
+    /// <typeparam name="T">Object type</typeparam>
+    /// <typeparam name="L">Language</typeparam>
+    public class LocalizedPropertyByName<T, L>
+    {
+        private object _propertyValue;
+
+        /// <summary>
+        /// Ctor
+        /// </summary>
+        /// <param name="propertyName">Property name</param>
+        /// <param name="func">Feature property access</param>
+        /// <param name="ignore">Specifies whether the property should be exported</param>
+        /// <returns>A task that represents the asynchronous operation</returns>
+        public LocalizedPropertyByName(string propertyName, Func<T, L, Task<object>> func, bool ignore = false)
```

Comment:
Hi, @mhsjaber! I guess that we can add this ctor to the PropertyByName class instead of adding a new class. As a result, new duplicate methods will be unnecessary. For example, here we can add a new flag and update the current logic of the PropertyManeger class to use language for this instance. What do you think about it?